intro.tex

```diff
@@ -1,41 +1,54 @@
 \section{Introduction}
 
-This  note describes the plan for ensuring the Rubin community will have sufficient data access, data products, and data analysis tools to produce early science when the full Legacy Survey of Space and Time (LSST)  begins. The start date for full survey operations is planned for April 1, 2024 with considerable uncertainty. As of November 2021, the construction project schedule shows completetion between December 2023 and July 2024 (i.e. planned finish and finish with contingency). It is unlikely the project will finish in December 2023 and highly likely it will finish by July 2024. 
+This  note describes the plan for ensuring the Rubin community will have the data products and services necessary to produce early science when the full Legacy Survey of Space and Time (LSST)  begins. 
```

Comment:
maybe 
... produce early science during the time between commissioning and the first data release in year 1.